geometry_msgs/Point32[2] buoyLocations
std_msgs/String[2] buoyColors
std_msgs/Int32 gateNumber
geometry_msgs/Pose2D midpointAndBearing
